Resources/Prototypes/_LostParadise/Maps/Pools/LostParadiseMapPool.yml

```
@@ -1,9 +1,20 @@
 - type: gameMapPool
   id: LostParadiseMapPool
   maps:
-  - LPPAsterisk
+#  - LPPAsterisk
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)